src/iotjs_module_buffer.h

```diff
@@ -42,16 +42,16 @@ class BufferWrap : public JObjectWrap {
   JObject& jbuiltin();
   JObject& jbuffer();
 
-  char* buffer();
+  jschar* buffer();
   size_t length();
 
   int Compare(const BufferWrap& other) const;
 
-  size_t Copy(char* src, size_t len);
-  size_t Copy(char* src, size_t src_from, size_t src_to, size_t dst_from);
+  size_t Copy(jschar* src, size_t len);
+  size_t Copy(jschar* src, size_t src_from, size_t src_to, size_t dst_from);
 
  protected:
-  char* _buffer;
+  jschar* _buffer;
```

Comment:
`Buffer` is for array of bytes, not for javascript string. Thus `_buffer` should be a type representing array of bytes.